### PR TITLE
Feature/session limit

### PR DESCRIPTION
**Pull request by anshulrr**
1. Added session limit for organization and API changes to update it.
2. Added session uuid for inbound messages

### CODE REVIEW

File: api.docs/includes/_organizations.md [hunk 1]
```diff
@@ -794,6 +802,11 @@ Unique
 <td valign="top"><a href="#string">String</a></td>
 <td></td>
 </tr>
+<tr>
+<td colspan="2" valign="top"><strong>sessionLimit</strong></td>
+<td valign="top"><a href="#integer">Integer</a></td>
```
Comment (pankaj-ag) [inline api.docs/includes/_organizations.md:807]: Let's mentioned that it's in minutes. 
```suggestion
<td valign="top"><a href="#integer">Integer (in minutes)</a></td>
```

File: lib/glific/messages.ex [hunk 2]
```diff
@@ -759,4 +759,33 @@ defmodule Glific.Messages do
...
+             organization.session_limit,
+         false <-
+           is_nil(
+             last_inbound_message =
+               Message
+               |> where([m], m.contact_id == ^contact.id)
+               |> where([m], m.flow == "inbound")
+               |> order_by([m], desc: m.id)
+               |> first()
+               |> Repo.one()
```
Comment (pankaj-ag) [inline lib/glific/messages.ex:783]: Why we need both functions  |> first and Repo.one ?

File: lib/glific/messages.ex [hunk 3]
```diff
@@ -759,4 +759,33 @@ defmodule Glific.Messages do
...
   end
+
+  @doc """
+  Get session uuid of contact's last inbound message
+  """
+  @spec get_session_uuid(map()) :: Ecto.UUID.t()
+  def get_session_uuid(message_params) do
+    organization = Partners.organization(message_params.organization_id)
+
+    with {:ok, contact} <-
```
Comment (pankaj-ag) [inline lib/glific/messages.ex:770]: I think we already have a function called get organization contact which avoided the DB call?

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific/messages.ex (hunk 2 above)
Comment (anshulrr) [reply lib/glific/messages.ex:783]: We need |> first to create the query, and then Repo.one() or Repo.all() to fetch it.

File: lib/glific/messages.ex (hunk 3 above)
Comment (anshulrr) [reply lib/glific/messages.ex:770]: We might remove this code and avoid DB call when switching to trigger.

**Pull request merged**